spec/requests/stories/feeds_spec.rb

```diff
@@ -114,6 +114,17 @@
       end
     end
 
+    context "when there are highly rated comments" do
+      let(:comment) { create(:comment, score: 20) }
+      let(:article) { comment.commentable }
```

Comment:
What does the `commentable` property represent? Is this related to the "Hide" comment feature?

Reply:
It's the link back to `article` on the `comment`. A comment can be attached to several different classes (`article`, `podcast_episode`, or `user`, that's why it's generalized here